applications_sbx/analytics/backend.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: backend
+  namespace: argocd
+  labels:
+    purpose: analytics
+spec:
+  destination:
+    namespace: analytics
+    server: https://kubernetes.default.svc
+  project: default
+  source:
+    path: ionos_sbx/analytics/backend
```

Comment:
path: ionos_sbx/analytics/backend

this does not exist under ionos_sbx, please make the needed changes